recipes/get_version/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "get_version" %}
+{% set version = "2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url:  https://github.com/flying-sheep/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: ed320df3b0604ecab925f5e00290c09f85c169bc8f826bb263b0d82889be487c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m flit install --deps none"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
```

Comment:
Do you need `pip` if you're using `flit`?
```suggestion
```

Reply:
`pip` is a dependency of `flit`, so no, not explicitly.